### PR TITLE
Http service

### PR DESCRIPTION
**Pull request by Gaddsky**
Please, check whether I code it in right way

### CODE REVIEW

File: fx-service/src/main/java/fx/service/Account.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package fx.service;
+
+/**
+ * <a href="http://developer.oanda.com/rest-live/accounts/">
+ *     Information about user account
+ * </a>
+ */
+public class Account {
```
Comment (pasby) [inline fx-service/src/main/java/fx/service/Account.java]: AccountsService - good name for class

File: fx-service/src/main/java/fx/service/Account.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    /**
+     * <a href="http://developer.oanda.com/rest-live/accounts/#getAccountsForUser">
+     *     Get a list of accounts owned by the user
+     * </a>
+     * @return fx.domain.Accounts
+     * @throws ServiceHttpException
+     */
+    public static fx.domain.Accounts getAll() throws ServiceHttpException {
```
Comment (pasby) [inline fx-service/src/main/java/fx/service/Account.java]: no static methods

File: fx-service/src/main/java/fx/service/Account.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        HttpBuilder hb = new HttpBuilder("https://api-fxpractice.oanda.com/v1/accounts");
+        Response resp = hb.header("Authorization", "Bearer " + System.getenv("oandatoken")).get().execute();
```
Comment (pasby) [inline fx-service/src/main/java/fx/service/Account.java]: make token pivate field and init it in init() method

File: fx-service/src/main/java/fx/service/Account.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+    /**
+     * <a href="http://developer.oanda.com/rest-live/accounts/#getAccountInformation">
+     *     Get account information
+     * </a>
+     * @param accountID id of user account (can be obtained with getAll() method)
+     * @return fx.domain.AccountInformation
+     * @throws ServiceHttpException
+     */
+    public static fx.domain.AccountInformation getAccountInformation(int accountID) throws ServiceHttpException {
```
Comment (pasby) [inline fx-service/src/main/java/fx/service/Account.java]: use imports

File: fx-service/src/main/java/fx/service/Account.java (hunk 3 above)
Comment (Gaddsky) [inline fx-service/src/main/java/fx/service/Account.java]: Not understand, what you mean. I did another solution - I implement method authorization() in HttpBuilder. If this method is in the HttpBuilder, that token is added to request.

**Pull request merged**